crypto/mem.c

```diff
@@ -14,6 +14,19 @@
 #include "internal/cryptlib.h"
 
 /*
+* allow Visual Studio 2003 and higher dump possible memory leaks with
+* _CrtDumpMemoryLeaks() API if OPENSSL_NO_CRYPTO_MDEBUG is defined and library
+* local memory debugging functionality is disabled
+*/
+#if defined(OPENSSL_NO_CRYPTO_MDEBUG) && defined(_DEBUG) && \
+    defined(_MSC_VER) && (_MSC_VER >= 1310)
+# include <crtdbg.h>
+# ifdef _NORMAL_BLOCK
+   #define MSVC_MDEBUG
```

Comment:
```
#    define MSVC_DEBUG
```
It's also
```
/*
 *
 */
```
i.e. stars in comment get to be aligned.

As already mentioned add `[skip ci]` to reformat commit messages...